client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
Ok, let's see. 

In which way is that envisioned to be used? Or in other words, what's the definition/meaning of that keyword?